internal/innosat/telemetry.go

```diff
@@ -0,0 +1,30 @@
+package innosat
+
+import (
+	"encoding/binary"
+	"io"
+)
+
+//TMDataFieldHeader (9 octets)
+type TMDataFieldHeader struct {
+	PUS             uint8
+	ServiceType     uint8
+	ServiceSubType  uint8
+	CUCTimeSeconds  uint32
+	CUCTimeFraction uint16
+}
+
+// Read TMDataFieldHeader
+func (tmdfh *TMDataFieldHeader) Read(buf io.Reader) error {
+	return binary.Read(buf, binary.BigEndian, tmdfh)
```

Comment:
how about a test of a real TM Data Field Header buffer from a `.rac` file resulting in the expected struct?